### PR TITLE
Specify limiting number of aggregatable reports per source

### PR DESCRIPTION
**Pull request by linnan-github**
Spec for https://github.com/WICG/attribution-reporting-api/pull/774

***
<a href="https://pr-preview.s3.amazonaws.com/linnan-github/conversion-measurement-api/pull/778.html" title="Last updated on May 3, 2023, 7:51 PM UTC (6b5c895)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/attribution-reporting-api/778/c83fa51...linnan-github:6b5c895.html" title="Last updated on May 3, 2023, 7:51 PM UTC (6b5c895)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -761,6 +764,9 @@ controls how many times a single [=attribution source=] whose
 [=attribution source/source type=] is "<code>[=source type/event=]</code>" can create an
 [=event-level report=].
 
+<dfn>Max aggregatable reports per source</dfn> is a positive integer that controls how many times a
+single [=attribution source=] can create an [=aggregatable report=].
```
Comment (apasel422) [inline index.bs:768]: I might phrase this differently, as the source doesn't really create the report. The report is rather the result of attribution.

**Review by apasel422 — DISMISSED** [review]

File: header-validator/validate-json.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const int64Regex = /^-?[0-9]+$/
 const hex128Regex = /^0[xX][0-9A-Fa-f]{1,32}$/
 
 const limits = {
-  maxAggregationKeys: 50,
+  maxAggregationKeys: 20,
```
Comment (apasel422) [inline header-validator/validate-json.js:6]: Could you clarify why we're making this change in this PR?

Comment (linnan-github) [reply header-validator/validate-json.js:6]: Actually this is merged from another branch by mistake, will revert.

File: index.bs (hunk 1 above)
Comment (linnan-github) [reply index.bs]: Rephrased a little bit, wdyt? Thanks.

**Review by csharrison — APPROVED** [review]

**Review by apasel422 — APPROVED** [review]

**Pull request merged**